frontend/app/admin/src/views/organizations/tags/EditOrganizationTagView.vue

```diff
@@ -11,47 +11,110 @@
+                </template>
+            </STList>
+            <p>
+                <button class="button text" type="button" @click="selectTags">
+                    <span class="icon add" />
+                    <span>{{ isEmpty ? $t('Voeg bestaande tags toe') : $t('Wijzig tags') }}</span>
+                </button>
+            </p>
+        </div>
+
+        <div v-if="!isNew" class="container">
             <hr>
             <h2>
-                Verwijder deze tag
+                {{ $t('Verwijder deze tag') }}
             </h2>
+            <p v-if="!isEmpty" class="style-description">
+                {{ $t('De tags in deze categorie zullen niet verwijderd worden.') }}
+            </p>
 
             <button class="button secundary danger" type="button" @click="doDelete">
                 <span class="icon trash" />
-                <span>Verwijderen</span>
+                <span>{{ $t('Verwijderen') }}</span>
             </button>
         </div>
     </SaveView>
 </template>
 
 <script setup lang="ts">
-import { AutoEncoderPatchType } from '@simonbackx/simple-encoding';
+import { AutoEncoderPatchType, PatchableArray } from '@simonbackx/simple-encoding';
 import { SimpleError } from '@simonbackx/simple-errors';
-import { usePop } from '@simonbackx/vue-app-navigation';
-import { CenteredMessage, ErrorBox, SaveView, useErrors, usePatch } from '@stamhoofd/components';
+import { ComponentWithProperties, usePop, usePresent } from '@simonbackx/vue-app-navigation';
+import { CenteredMessage, ErrorBox, OrganizationTagSelectorView, SaveView, useDraggableArrayIds, useErrors, usePatch, usePlatform } from '@stamhoofd/components';
 import { OrganizationTag } from '@stamhoofd/structures';
-import { computed, ref } from 'vue';
+import { computed, ref, UnwrapNestedRefs } from 'vue';
+import TagRow from './components/TagRow.vue';
+import EditOrganizationTagView from './EditOrganizationTagView.vue';
 
 const errors = useErrors();
+const present = usePresent();
 const saving = ref(false);
 const deleting = ref(false);
 
 const props = defineProps<{
     tag: OrganizationTag;
+    allTags: OrganizationTag[];
     isNew: boolean;
-    saveHandler: (p: AutoEncoderPatchType<OrganizationTag>) => Promise<void>;
-    deleteHandler: (() => Promise<void>) | null;
+    saveHandler: (patches: { tag: OrganizationTag; patch: AutoEncoderPatchType<OrganizationTag> }[]) => Promise<void>;
+    deleteHandler: ((tagIds: string[]) => Promise<void>);
 }>();
+
 const title = computed(() => props.isNew ? 'Nieuwe tag' : 'Tag bewerken');
 const pop = usePop();
+const platform = usePlatform();
+
+const { patched, addPatch, hasChanges: hasTagChanges, patch } = usePatch(props.tag);
+
+const deletedTags = ref<string[]>([]);
+type UseChildPatchMapValue = UnwrapNestedRefs<ReturnType<typeof usePatch<OrganizationTag>>>;
+const useChildPatchMap = ref<Map<string, UseChildPatchMapValue>>(new Map());
```

Comment:
De manier van patches bijhouden is te ingewikkeld en te custom.

In plaats daarvan moet je defaulten naar één single source of truth (er moeten alarmlichten afgaan vanaf een view meer dan één patch heeft). In dit geval is dat `PatchableArrayAutoEncoder<OrganizationTag>`.

Pseudo code:
```
const patch: PatchableArrayAutoEncoder<OrganizationTag> = new PatchableArray();
const allPatchedTags = props.allTags.patch(patch);
const patched = allPatchedTags.find(t => t.id === props.tag.id);
```

De savehandler moet ook best gewoon één argument hebben die PatchableArrayAutoEncoder. De deletehandler kan daardoor ook verdwijnen en via dezelfde save handler lopen.